docs/api/en/textures/Data3DTexture.html

```diff
@@ -26,30 +29,115 @@ <h3>[name]( [param:TypedArray data], [param:Number width], [param:Number height]
 			[page:Number depth] -- depth of the texture.
 		</p>
 
+		<h2>Code Example</h2>
+
+		<p>This creates a [name] with repeating data, 0 to 63</p>
+
+		<code>
+		// create a buffer with some data
+
+		const sizeX = 64;
+		const sizeY = 64;
+		const sizeZ = 64;
+
+		const data = new Uint8Array( sizeX * sizeY * sizeZ );
+
+		let i = 0;
+
+		for ( let z = 0; z < sizeZ; z ++ ) {
+
+			for ( let y = 0; y < sizeY; y ++ ) {
+
+				for ( let x = 0; x < sizeX; x ++ ) {
+
+					data[ i ] = i mod 64;
```

Comment:
> i mod 64

Have you tested this code? I does not look like valid JavaScript...

I'm not sure a code example is required at all. We have one for normal data texture and I think devs should be example to expand it to the 3D case by themselves.

Reply:
Modulo operator ( % ) to ensure number is well in range of Uint8 during assignment.  Updated to be example.

Reply:
Then code is still wrong. The value range of `Uint8Array` is `[0, 255]`. It should be:
```js
data[ i ] = i % 256;
```

Reply:
I'll update it from [0, 63] to [0, 255].  I just chose a number I could calculate values for in my head for unit tests on 64 cube.

Would look more colorful if rendered. Random gets floored to 0 so can't use it for testing.

Reply:
I though about this:
```js
data[ i ] = THREE.MathUtils.randInt( 0, 255 );
```
But never mind. Let's keep the modulo computation.

Reply:
Done.  Thank you.